src/services/codex-coin.js

```diff
@@ -0,0 +1,17 @@
+// import { contracts } from '@codex-protocol/ethereum-service'
+//
+// import config from '../config'
+// import models from '../models'
+import SocketService from './socket'
+
+export default {
+
+  transfer(fromAddress, toAddress, value, transactionHash) {
```

Comment:
I question the value in listening to these events just for the purposes of emitting an event since that can all be done natively with web3js. Are you planning on tracking user balance + approval state in the db too?